ci/scripts/create-and-update-db.sh

```diff
@@ -33,16 +33,24 @@ if ! contains "$DATABASES" "$db_user"; then
   DATABASES="$db_user $DATABASES"
 fi
 
+
+
 for db in ${DATABASES}; do
 
   # Create database
-  psql_adm -d postgres -l | awk '{print $1}' | grep -q "${db}" || \
-    psql_adm -d postgres -c "CREATE DATABASE ${db} OWNER ${db_user}"
-
-  # Enable extensions
-  for ext in citext uuid-ossp pgcrypto pg_stat_statements; do
-    psql_adm -d "${db}" -c "CREATE EXTENSION IF NOT EXISTS \"${ext}\""
-  done
+  if ! [ "${db}" = "credhub" ]; then
+    psql_adm -d postgres -l | awk '{print $1}' | grep -q "${db}" || \
+      psql_adm -d postgres -c "CREATE DATABASE ${db} OWNER ${db_user}"
+    # Enable extensions
+    for ext in citext uuid-ossp pgcrypto pg_stat_statements; do
+      psql_adm -d "${db}" -c "CREATE EXTENSION IF NOT EXISTS \"${ext}\""
+    done
+  elif [ "${db}" = "credhub"]; then
```

Comment:
We don't need to do this sort of check. We should just have the `"${db}" = credhub` check and everything else gets the default which is the extension creation.